vllm/model_executor/models/dbrx.py

```diff
@@ -86,17 +86,15 @@ def __init__(
                 self.num_total_experts,
                 2 * self.intermediate_size,
                 self.d_model,
-                device="cuda",
                 dtype=self.params_dtype,
-            ))
+            ), requires_grad=False)
```

Comment:
any reason we add requires_grad=False here? shouldn't it be false by default?

Reply:
@kzawora-intel , no specific reason, I saw other models also explicitly put 'requires_grad=False', so I also added here.
Do you want me to remove this?